industrial_robot_client/src/joint_trajectory_streamer.cpp

```diff
@@ -167,7 +168,7 @@ void JointTrajectoryStreamer::streamingThread()
     switch (this->state_)
     {
       case TransferStates::IDLE:
-        ros::Duration(0.250).sleep();  //  slower loop while waiting for new trajectory
+        ros::Duration(0.1).sleep();  //  slower loop while waiting for new trajectory
```

Comment:
No a big deal, but I'm curious why you needed/wanted a faster loop.

Reply:
At some point we were having timing issues with the Motoman driver and this was probably done on site in that debug.  Again, I apologize for having Croft like this in the PR.  I can submit a new one with only the relevant changes if that makes your life easier.

Reply:
Some of this cruft may have happened after the original PR was submitted, but yeah, that’s what feature branches are for.  Mid-2017 was hectic so I can spend some time cleaning this up properly if needed.

Reply:
I recall now.  In our production cell we send probably 30 trajectories per task cycle.  This is a time critical application with a hard 60 second cycle time.  A 0.25 loop meant a possible worst-case delay of 7.5s over the 30 trajectories.  7.5s versus 3s (at 0.1 s loop) bought us 4.5s per cycle best case.  In practice it was less but still noticeable.